contribs/gmf/src/controllers/abstractmobile.js

```diff
@@ -95,14 +112,12 @@ gmf.AbstractMobileController = function(ngeoFeatureOverlayMgr, serverVars) {
    * @export
    */
   this.map = new ol.Map({
-    layers: [
-      new ol.layer.Tile({
-        source: new ol.source.OSM()
-      })
-    ],
+    layers: [],
     view: new ol.View({
-      center: [0, 0],
-      zoom: 2
+      center: [632464, 185457],
+      projection: ol.proj.get('epsg:21781'),
```

Comment:
How did you choose that ? I'm not sure about this as well, this should be overwritten by config some how.